igvm/hypervisor.py

```diff
@@ -555,9 +556,23 @@ def vm_block_device_name(self):
         the guest OS"""
         return 'vda1'
 
+    def migrate_wait_shutdown(
+        self, vm: VM, no_shutdown: bool, transaction: Transaction,
+    ):
```

Comment:
This should method should be a parameter of `migrate_vm`.

Reply:
Don't get it. What actually should be a parameter? All of the arguments of this method are parameters of `migrate_vm`

Reply:
My bad I missed how it is used in migrate_vm. This is fine.

You could still add a doc string and consider making it a underscore method.